app/helpers/application_helper/toolbar/container_group_center.rb

```diff
@@ -58,6 +58,11 @@ class ApplicationHelper::Toolbar::ContainerGroupCenter < ApplicationHelper::Tool
           'pficon pficon-edit fa-lg',
           N_('Edit Tags for this #{ui_lookup(:table=>"container_group")}'),
           N_('Edit Tags')),
+        button(
+          :container_group_protect,
+          'pficon pficon-edit fa-lg',
+          N_('Manage Policies for this #{ui_lookup(:table=>"container_group")}'),
+          N_('Manage Policies')),
```

Comment:
> But what does it mean for these entities?

the protect button take you to a screen where you assign policies. the compliance button performs a compliance check - i'm pretty sure what it does is takes all the policies currently assigned to that object of type compliance and evaluates them.
(behind the scenes  a compliance event is raised to that object)

since the separation added some additional logic I would like to see compliance here as well